recipes-core/systemd/systemd/229/0001-nspawn-don-t-check-binary-OS-tree-before-all-mounts-.patch

```diff
@@ -0,0 +1,75 @@
+From 0a95296b24d636377a58592f1a30eb4312e5d3d2 Mon Sep 17 00:00:00 2001
+From: Krisztian Litkey <krisztian.litkey@intel.com>
+Date: Tue, 8 Dec 2015 16:02:56 +0200
+Subject: [PATCH 1/2] nspawn: don't check binary/OS-tree before all mounts are
+ in place.
+
+Do not check if the binary to run is in place or if the given
+directory looks like an OS tree hierarchy *before* we have put
+all the requested bind- and overlay-mounts in place. It is
+possible that some of the mounts are responsible for pulling in
+the necessary bits and pieces for a successful check.
+
+Upstream-Status: Pending
```

Comment:
you can merge it... I'm okay, it was mostly a "+0". You can add the links later.